amads/algorithms/norm.py

```diff
@@ -0,0 +1,208 @@
+"""
+Local implementation of standard normalization routines centrally for use in various functions.
+The local implementation serves to obviate the need for external libraries (e.g., scipy) in basic cases.
+
+Comparisons on this module support any pair of profiles.
+Example use cased include pitch class profile matching for 'best key' measurement,
+and the metrical equivalent.
+"""
+
+from typing import Iterable
+
+import numpy as np
+
+__author__ = "Mark Gotham"
+
+
+# ------------------------------------------------------------------------------
+
+l1_names = ["sum", "manhattan", "city", "cityblock", "city block", "taxicab", "l1"]
```

Comment:
It's nice that you provide these options but I think more conventionally you would just give one name and then the user can work out this name by looking at the documentation!